apps/web/src/components/FormInstance.tsx

```diff
@@ -90,265 +75,130 @@ const FormInstance = ({
-                  : 'var(--mfa-blue-hex)'
-              }
-              color="#FFF"
-              onClick={async (_) => {
-                toast.promise(_handleFormSign(), {
-                  success: {
-                    title: 'Success',
-                    description: 'Form signed',
-                  },
-                  error: {
-                    title: 'Error',
-                    description: 'Unable to sign form',
-                  },
-                  loading: {
-                    title: 'Pending',
-                    description: 'Please wait',
-                  },
-                });
-              }}
-            >
-              Sign Form
-            </Button>
-          )}
-          {formInstance.completed &&
-            user?.id === formInstance.originator.id && (
-              <Box>
-                <Spacer />
-                <Button
-                  width="111px"
-                  height="40px"
-                  onClick={async (_) => {
-                    toast.promise(_handleFormApprove(), {
-                      success: {
-                        title: 'Success',
-                        description: 'Form approved',
-                      },
-                      error: {
-                        title: 'Error',
-                        description: 'Unable to approve form',
-                      },
-                      loading: {
-                        title: 'Pending',
-                        description: 'Please wait',
-                      },
-                    });
-                  }}
-                  background={
-                    formInstance.markedCompleted
-                      ? 'var(--mfa-gray-hex)'
-                      : 'var(--mfa-blue-hex)'
-                  }
-                  color="#FFF"
-                  cursor={
-                    formInstance.markedCompleted ? 'not-allowed' : 'pointer'
-                  }
-                >
-                  Approve
-                </Button>
-              </Box>
-            )}
         </Box>
       </Grid>
+
+      {_userCanSign && (
+        <Flex justifyContent="flex-end" mt={4}>
+          <Button
+            background={
+              formInstance.markedCompleted
+                ? 'var(--mfa-gray-hex)'
+                : 'var(--mfa-blue-hex)'
+            }
+            color="white"
+            onClick={async () => {
+              toast.promise(_handleFormSign(), {
+                success: {
+                  title: 'Success',
+                  description: 'Form signed',
+                },
+                error: {
+                  title: 'Error',
+                  description: 'Unable to sign form',
+                },
+                loading: {
+                  title: 'Pending',
+                  description: 'Please wait',
+                },
+              });
+            }}
+          >
+            Sign Form
+          </Button>
+        </Flex>
+      )}
     </Box>
   );
```

Comment:
Two other small spacing things -- I think the width of the pop out goes out less in the design and maybe the pdf editor is smaller?  Because I don't *think* the page scrolls in the design.

Reply:
This stuff would probably only take a couple minutes to fix, I could do it too lmk